connectors/automation-anywhere/element-templates/automation-anywhere-connector.json

```
@@ -0,0 +1,692 @@
+  },
+  "documentationRef": "https://docs.camunda.io/docs/components/modeler/web-modeler/connectors/available-connectors/automation-anywhere/",
+  "category": {
+    "id": "connectors",
+    "name": "Connectors"
+  },
+  "appliesTo": [
+    "bpmn:Task"
+  ],
+  "elementType": {
+    "value": "bpmn:ServiceTask"
+  },
+  "groups": [
+    {
+      "id": "operation",
+      "label": "Operation"
+    },
+    {
+      "id": "authentication",
+      "label": "Authentication"
+    },
+    {
+      "id": "configuration",
+      "label": "Configuration"
+    },
+    {
+      "id": "input",
+      "label": "Input"
+    },
+    {
+      "id": "output",
+      "label": "Output"
+    },
+    {
+      "id": "errors",
+      "label": "Error Handling"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "value": "io.camunda:http-json:1",
+      "binding": {
+        "type": "zeebe:taskDefinition:type"
+      }
+    },
+    {
+      "label": "Operation type",
+      "id": "operationType",
+      "group": "operation",
+      "value": "crateWorkItemModel",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "Create Work Item model",
+          "value": "createWorkItemModel"
+        },
+        {
+          "name": "Create queues",
+          "value": "createQueues"
+        },
+        {
+          "name": "Add Work Items to the queue",
+          "value": "addWorkItemsToTheQueue"
+        },
+        {
+          "name": "Update the Work Item by Id",
```

Please check with @bastiankoerber whether you need this operation.

```suggestion
          "name": "Update the Work Item by ID",
```